ENGAGEHF/Questionnaire/QuestionnaireSheetView.swift

```diff
@@ -14,15 +14,18 @@ import SwiftUI
 
 
 struct QuestionnaireSheetView: View {
-    private let questionnaireId: String
-    
-    @State private var questionnaire: Questionnaire?
-    @State private var viewState: ViewState = .idle
+    // MARK: - Type Properties
```

Comment:
Generally a good idea, but I think we probably want to stick to the current style in the App without Marks except for larger types that need an organization around functionality.